### ISSUE TITLE
MONDO:0019804 tracheomalacia

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#8506]
**Mondo term (ID and Label):**
MONDO:0019804 tracheomalacia

**Suggested new label:**
Congenital tracheomalacia

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**_Optional_: Any additional information (like supporting evidence, PubMed ID, etc.)**
Based on the use of the Orphanet definition, I don't think it's appropriate to call this term Tracheomalacia. Also Tracheomalacia is a phenotype in HPO, so I don't think it's really needed in Mondo: https://hpo.jax.org/browse/term/HP:0002779 
I don't know if you have a way to clarify with DO what they mean by their term DOID:0060313 (which uses these two terms as exact matches, it seems)

**Comment by nicolevasilevsky** [comment]
@kanems I relabeled the term to congenital tracheomalacia and made tracheomalacia a related synonym.

Let me know if any other action is needed, thanks!
